app/models/models_campaign.py

```diff
@@ -45,6 +45,12 @@ def as_dict(self):
         return {c.name: getattr(self, c.name) for c in self.__table__.columns}
 
 
+class VoterGroups(Base):
```

Comment:
I'm not sure `VoterGroups` is explicit enough for someone who does not know the module, to be honest.

Reply:
I don't know why we put plural for every model in this file, but we usually use singular names.
```suggestion
class VoterGroup(Base):
```

Reply:
> I'm not sure `VoterGroups` is explicit enough for someone who does not know the module, to be honest.

I'll add comments. No better name found for now

Reply:
> I don't know why we put plural for every model in this file, but we usually use singular names.

To avoid useless migrations, i would prefer staying with plural for every model in this file